kimchi/src/circuits/argument.rs

```diff
@@ -115,9 +114,9 @@ pub struct ArgumentData<F: 'static> {
 /// Witness data for a argument
 pub struct ArgumentWitness<T> {
     /// Witness for current row
-    pub curr: [T; COLUMNS],
+    pub curr: Vec<T>,
```

Comment:
Convert to a slice, or make generic over columns?

Reply:
Oh thank you, I don't know how I missed that one 😮